src/components/atoms/Icon/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './style.css';
+
+const Icon = ({ src, alt, onClick }) => {
+    return (
+        <img className="icon" src={src} alt={alt} onClick={onClick} />
```

Comment:
why icon should have onclick?

Reply:
Why not? I use icons like buttons for drop-down menu.

Reply:
why not to use buttons?

Reply:
What happens if click on img?